include/fem-dem/gls_sharp_navier_stokes.h

```diff
@@ -362,10 +363,26 @@ class GLSSharpNavierStokesSolver : public GLSNavierStokesSolver<dim>
   cell_cut(const typename DoFHandler<dim>::active_cell_iterator &cell,
            std::vector<types::global_dof_index> &         local_dof_indices,
            std::map<types::global_dof_index, Point<dim>> &support_points);
+
+  /**
+   * @brief
+   * Return a bool to define if a cell is cut by an IB particle and the local
+   * DOFs of the cell for later use. This function is built to handle special
+   * cases where the level set is always positive.
```

Comment:
I don't understand this part. This function does not return the local DOFs of the cell. That brief block is tough to follow.